packages/flutter/lib/src/material/page.dart

```diff
@@ -67,21 +67,17 @@ class MaterialPageRoute<T> extends PageRoute<T> {
     @required this.builder,
     RouteSettings settings: const RouteSettings(),
     this.maintainState: true,
-    this.fullscreenDialog: false,
-  }) : super(settings: settings) {
-    assert(builder != null);
-    assert(opaque);
-  }
+    bool fullscreenDialog: false,
+  }) :
+      _fullscreenDialog = fullscreenDialog,
```

Comment:
A good example of another case where we regularly indent an odd number of spaces is multiline expressions on `return` statements. Having pretty lined-up expressions and no weird line breaks far outweighs any ugliness from having odd numbers of spaces at the start of lines... I'm not sure I'd even notice the odd number of spaces thing. :-)